src/tests/internal/test_evaluators.py

```diff
@@ -108,6 +108,8 @@ def test_get_happy_evaluator():
     assert isinstance(evaluator, HappyEvaluator)
 
 
+
+
```

Comment:
Not super important but: you can add a short test that ensures Evaluator.LLM_JUDGE_AZURE_GPT is an LLMJudgeEvaluator.

Reply:
Added test for checking Evaluator type.